### PR TITLE
Fix swin dataloader import bug

### PR DESCRIPTION
**Pull request by BBuf**
/home/zhangxiaoyu/oneflow/python/oneflow/__init__.py
/home/zhangxiaoyu/.local/lib/python3.8/site-packages/flowvision/transforms/transforms.py:745: UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
  warnings.warn(
/home/zhangxiaoyu/.local/lib/python3.8/site-packages/torch/__init__.py
OneFlow swin dataloader time: 0.005s (= 0.918s / 200, num_workers=2)
PyTorch swin dataloader time: 0.003s (= 0.539s / 200, num_workers=2)
Relative speed: 0.587 (= 0.003s / 0.005s)

### CODE REVIEW

File: scripts/swin_dataloader_compare_speed_with_pytorch.py [hunk 1]
```diff
@@ -64,99 +57,104 @@ def ensure_dataset():
...
-    total_time = time.time() - start_time
-    return total_time
+    else:
+        import oneflow as flow
+        from oneflow.utils.data import DataLoader
+
+        from flowvision import datasets, transforms
+        from flowvision.data import create_transform
+
+    class SubsetRandomSampler(flow.utils.data.Sampler):
```
Comment (daquexian) [inline scripts/swin_dataloader_compare_speed_with_pytorch.py:84]: 这个是不是可以去掉，在创建 DataLoader 的时候指定 shuffle=True 就可以了

Comment (BBuf) [reply scripts/swin_dataloader_compare_speed_with_pytorch.py:84]: 好的

**Review by daquexian — APPROVED** [review]
